binstar_client/commands/notebook.py

```diff
@@ -103,17 +104,14 @@ def add_download_parser(subparsers):
 def upload(args):
     binstar = get_binstar(args)
     uploader = Uploader(binstar, args.notebook, user=args.user, summary=args.summary,
-                        version=args.version)
+                        version=args.version, thumbnail=args.thumbnail, name=args.name)
 
-    if os.path.exists(args.notebook):
+    try:
         upload_info = uploader.upload(force=args.force)
         log.info("{} has been uploaded.".format(args.notebook))
         log.info("You can visit your notebook at {}".format(notebook_url(upload_info)))
-    else:
-        raise errors.BinstarError("{} can't be found".format(args.notebook))
-
-    if uploader.msg is not None:
-        log.error(uploader.msg)
+    except (errors.BinstarError, IOError) as e:
```

Comment:
What is the purpose of this try catch? `errors.BinstarError` are already caught and logged at a higher level.